packages/web-components/src/appointment-planner/server/serverless.yml

```diff
@@ -0,0 +1,90 @@
+service: web-components.appointment-planner
```

Comment:
Please change the service name to `web-components-appointment-planner`, because `.` is not allowed

Reply:
In serverless.yml of search-widget we have `service: web-components.search-widget` so why web-components.appointment-planner is not allow ?

Reply:
![image](https://user-images.githubusercontent.com/22660833/82819421-8cbcfe80-9eca-11ea-8ba7-0a42d21ce8fd.png)
I think should change to web-components-appointment-planner for consist with existed one

Reply:
@khacvy007 It's not allowed and I had to change it, without `.` https://github.com/reapit/foundations/blob/master/packages/web-components/src/search-widget/server/serverless.yml#L1